addons/account/models/account_report.py

```diff
@@ -222,33 +222,32 @@ def write(self, vals):
 
         return super().write(vals)
 
+    def copy_data(self, default=None):
+        vals_list = super().copy_data(default=default)
+        return [dict(vals, name=report._get_copied_name()) for report, vals in zip(self, vals_list)]
```

Comment:
Same issue as other places, it can fail to deduplicate the name, but I suppose it is an acceptable limitation.
Easy to fix with a cache though.